### PR TITLE
Add Report Files

### LINKED ISSUE #30: Create controllers and views for reports.
# REQUIREMENTS
Create controllers for Reports.
Create views for Reports.

# Acceptance Criteria

- [ ] Created controllers.
- [ ] Created views.

### PR DESCRIPTION
**Pull request by SampabloC**
We add controllers and views for Reports.
Modify routes.rb to add resources for reports.
Add some buttons in report index view to link to other class.

Co-authored-by: RobertoCarMed <betocarrasco3652@gmail.com>

# Description
- We add controllers and views for Reports.
- Modify routes.rb to add resources for reports.
- Add some buttons in report index view to link to other class.

Co-authored-by: RobertoCarMed <betocarrasco3652@gmail.com>

Resolve #30 

  ## Screenshots 
Report index view
imagen

### CODE REVIEW

File: app/helpers/reports_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module ReportsHelper
+end
```
Comment (xFerchoVJ) [inline app/helpers/reports_helper.rb:2]: If you don't need it you can delete it

**Review by SaraNavarro — DISMISSED** [review]
Good :)

**Review by xFerchoVJ — DISMISSED** [review]
LGTM!

File: app/helpers/reports_helper.rb (hunk 1 above)
Comment (RobertoCarMed) [reply app/helpers/reports_helper.rb:2]: thanks, this resolving in next issue

File: app/views/reports/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<h1>Report View</h1>
+
+<div>
+  <%= link_to "Day", administrators_path, class: "btn btn-success" %>
+  <%= link_to "Month", administrators_path, class: "btn btn-success" %>
+  <%= link_to "Check In/Out Average", administrators_path, class: "btn btn-success" %>
+</div>
+<%= link_to "Back", administrators_path, class: "btn btn-info" %>
```
Comment (YilKanda) [inline app/views/reports/index.html.erb]: Add a blank line

File: app/assets/stylesheets/reports.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Reports controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (YilKanda) [inline app/assets/stylesheets/reports.scss]: If you don't occupy it, delete it.

**Review by YilKanda — COMMENTED** [review]
I left you some comments, only related to the Style Guide, but in general it looks good.

**Review by xFerchoVJ — APPROVED** [review]
LGTM:)

**Review by YilKanda — APPROVED** [review]
Just as a recommendation:

Be more specific on commits.

Remember that commits must contain significant changes, and you must also detail in the commit what was done, and why it was done.

This will allow other team members to identify in the git commit tree what changes were made to your branch.

I comment, because I see two commits that say: 'Update", but it is not explained why the change was made, and what the change was.

When you have a commit made and you need to make changes within that commit, which are not significant (such as deleting an empty line or adding a blank line, etc.), you can make use of git amend.

You add the changes to the stage:

`git add -p`

or

`git add .`

Then you do the commit, but adding the amend:

`git  commit --amend`

This also allows you to change the content of the commit script.

And in case you don't want to change the content of your commit script, you commit with a --no-edit:

`git commit --amend --no-edit`

Lastly, you do a push, but add the --force-with-lease option:

`git push --force-with-lease origin name-of-your-branch`

Remember, one of the purposes of the course is to experiment with git. So don't be afraid to use these kinds of git statements.

Have a nice day!

**Review by SEVERO96 — APPROVED** [review]

**Pull request merged**